flyteadmin/pkg/repositories/config/config.go

```diff
@@ -0,0 +1,125 @@
+}
+
+type ClusterResourceAttributes struct {
+	Project    string            `json:"project" pflag:", The project to which the cluster resource attribute is applied."`
+	Domain     string            `json:"domain" pflag:", The domain to which the cluster resource attribute is applied."`
+	Workflow   string            `json:"workflow" pflag:", The workflow to which the cluster resource attribute is applied."`
+	Attributes map[string]string `json:"attributes" pflag:", The cluster resource attributes."`
+}
+
+type ExecutionQueueAttributesConfig struct {
+	Declarative bool                       `json:"declarative" pflag:", Whether the execution queue attributes are managed declaratively, i.e. whether non-listed attributes are deleted."`
+	Values      []ExecutionQueueAttributes `json:"values" pflag:", The list of execution queue attributes to be managed."`
+}
+
+type ExecutionQueueAttributes struct {
+	Project  string   `json:"project" pflag:", The project to which the execution queue attribute is applied."`
+	Domain   string   `json:"domain" pflag:", The domain to which the execution queue attribute is applied."`
+	Workflow string   `json:"workflow" pflag:", The workflow to which the execution queue attribute is applied."`
+	Tags     []string `json:"tags" pflag:", The tags of the execution queue."`
+}
+
+type PluginOverridesConfig struct {
+	Declarative bool              `json:"declarative" pflag:", Whether the plugin overrides are managed declaratively, i.e. whether non-listed overrides are deleted."`
+	Values      []PluginOverrides `json:"values" pflag:", The list of plugin overrides to be managed."`
+}
+
+type PluginOverrides struct {
+	Project  string `json:"project" pflag:", The project to which the plugin override is applied."`
+	Domain   string `json:"domain" pflag:", The domain to which the plugin override is applied."`
+	Workflow string `json:"workflow" pflag:", The workflow to which the plugin override is applied."`
+
+	Overrides []PluginOverride `json:"overrides" pflag:", The list of plugin overrides."`
+}
+
+type PluginOverride struct {
+	TaskType              string   `json:"task_type" pflag:", Task type for which to apply plugin implementation overrides."`
+	PluginId              []string `json:"plugin_id" pflag:", Plugin id(s) to be used in place of the default for the task type."`
+	MissingPluginBehavior int      `json:"missing_plugin_behavior" pflag:", Behavior when no specified plugin_id has an associated handler. 0 : FAIL , 1: DEFAULT."`
+}
+
+type WorkflowExecutionConfigConfig struct {
+	Declarative bool                      `json:"declarative" pflag:", Whether the workflow execution configs are managed declaratively, i.e. whether non-listed configs are deleted."`
+	Values      []WorkflowExecutionConfig `json:"values" pflag:", The list of workflow execution configs to be managed."`
+}
+
+type WorkflowExecutionConfig struct {
```

Comment:
This config override is not fully implemented yet because it has a lot of nested sub configs. Might do so in another PR to not overload this one.